cowdao_cowpy/cow/swap.py

```diff
@@ -93,7 +103,11 @@ def sign_order(chain: Chain, account: LocalAccount, order: Order) -> EcdsaSignat
         chain=chain, verifying_contract=CowContractAddress.SETTLEMENT_CONTRACT.value
     )
 
-    return _sign_order(order_domain, order, account, SigningScheme.EIP712)
+    sig = _sign_order(order_domain, order, account, SigningScheme.EIP712)
+    if not sig.data.startswith("0x"):
+        sig.data = "0x" + sig.data
```

Comment:
Is this required if you added

```
signature=signature.data if signature.data.startswith("0x") else f"0x{signature.data}",
```

below in the code?

Reply:
It doesnt seem to be, however it then looks to through a typing exception;

![image](https://github.com/user-attachments/assets/f370712f-1184-4c9d-9011-f0c203350cc1)


It seems to how for e2e order submission however!

Reply:
Hmmm
This causes some issues with typing and other checks post the syncing with main.

I added a new method here and hopfulyl cleaned up the ci a bit as well? 

However not sure if you would want to add it due to the e2e tests not being present on the main ci run.

https://github.com/8ball030/cow-py/pull/5